dragon/array_docs.rb

```diff
@@ -288,7 +288,7 @@ def tick args
     # with a width and height of 50.
     args.state
         .rainbow_colors
-        .map do |color| # <-- ~Array#each~ usage
+        .each do |color| # <-- ~Array#each~ usage
```

Comment:
Fix looks good to me. A side note is that the code would be better if it did use `map` like:

```
    # render sprites diagonally to the screen
    # with a width and height of 50.
    args.outputs.sprites << args.state.rainbow_colors.map do |color| #
          [
            color[:order] * 50,
            color[:order] * 50,
            50,
            50,
            "sprites/square-\#{color[:name]}.png"
          ]
    end
```

This results in 1 draw call instead of 7. The performance difference is negligible in this example; but I wish the example code used better performing code for sprite drawing as the standard.

Reply:
Yeah, there is an example of using `map` in the documentation too.  I assume this one was copy/pasted and the author just neglected to change this instance of `map` to `each`.